### ISSUE TITLE
Camera shows image from the last print.

### ISSUE THREAD

**Issue by acunyc** [AstroPrint/AstroBox#48]
It seems to be very common for the camera section (on the cloud and the box) to initially show the last image from the last print as the default image, until you take a new pic.

This issue appears to show up when the box has been on, a print completed (with images taken), then a new print is started (without the box being shutdown and restarted).

**Comment by acunyc** [comment]
New info.  I just played around with this more.  I printed a rocket, then printed a hook.  It took images of the rocket, then when I started the hook, it kept taking images of the hook, but they were still considered part of the rocket print capture.

So, now the rocket print capture shows the rocket being printed and then the hook.  I do not have a file for the hook print capture.

One possibility:  I hit the button to take a picture while it was heating up.  Perhaps this somehow caused a bypassing of ending the last print capture and creating a new one?

**Comment by acunyc** [comment]
Another possibility:  I was using the "each layer" feature.  Perhaps after the last layer it didn't know they print capture ended.

I've also noticed that a picture is not taken when the print finishes.  For example, I just printed a 5 layer rocket.  The last image is layer 4/5.  It would be good to have a final shot of the finished product.  Actually, it would probably be good to take 5-10 shots of the finished product.  That way when there is a print capture time lapse video it will show longer.
